assume/common/forecasts.py

```diff
@@ -82,6 +84,21 @@ def get_price(self, fuel_type: str) -> pd.Series:
 
         return self[f"fuel_price_{fuel_type}"]
 
+    def _to_fast_series(self, value, name: str) -> FastSeries:
```

Comment:
No, I meant to ask if they could be in utils. I see that this is currently just used in the forecaster. Yet this function seems very general, and it could be helpful in various locations in the future. If it is not, please indicate why.